MobileWallet/Common/BackgroundTasks/NodeSyncOperation.swift

```diff
@@ -48,7 +48,7 @@ class NodeSyncOperation: Operation {
             return
         }
 
-        NotificationManager.shared.scheduleNotification(title: "Background task", body: "Starting Tor")
+        //NotificationManager.shared.scheduleNotification(title: "Background task", body: "Starting Tor")
```

Comment:
Maybe remove this from code? or you plan to add it later?

Reply:
Need to use it later to debug background operations